README.md

```diff
@@ -1,24 +1,11 @@
-# README
+# Verify Support Form
```

Comment:
One thing missing which is to inform the user to set up the env vars for Zendesk. You could use something very similar to the `Prepare the application configuration` [here](https://github.com/alphagov/verify-rp-environment-config-form/blob/master/README.md).

Reply:
Ive mimicked the on-boarding form readme. Hope this is now ok?

Reply:
Perfect 👍